frontend/src/scenes/surveys/SurveyEditQuestionRow.tsx

```diff
@@ -164,6 +168,27 @@ export function SurveyEditQuestionGroup({ index, question }: { index: number; qu
                             const editingThankYouMessage =
                                 defaultSurveyFieldValues[question.type].appearance.thankYouMessageHeader !==
                                 survey.appearance?.thankYouMessageHeader
+
+                            const isNewTypeMultipleSurveyQuestion =
+                                newType === SurveyQuestionType.MultipleChoice ||
+                                newType === SurveyQuestionType.SingleChoice
+
+                            if (isCurrentTypeMultipleSurveyQuestion && isNewTypeMultipleSurveyQuestion) {
+                                setQuestionType(index, newType)
+                                resetBranchingForQuestion(index)
+                                return
+                            }
+                            if (isCurrentTypeMultipleSurveyQuestion && !isNewTypeMultipleSurveyQuestion) {
+                                if (
+                                    !confirm(
+                                        'The choices you have configured will be removed. Would you like to proceed?'
```

Comment:
Hey @benHPostHog, do you mind helping me out with this confirmation message? Would love to hear any suggestions of yours!

<img width="680" height="282" alt="Screenshot 2025-08-15 at 6 34 31 AM" src="https://github.com/user-attachments/assets/2a1654ae-b0d7-4b47-a769-b70ab742b9ce" />

Reply:
I updated the UI to use lemon dialog, let me know if you have any suggestions on the title and message :) 

<img width="1118" height="328" alt="Screenshot 2025-08-15 at 8 06 41 AM" src="https://github.com/user-attachments/assets/fe51043b-9c2a-4c8d-b6cf-0123c38d84cc" />